modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -1444,42 +1445,17 @@ public String toRegularExpression(String pattern) {
         return escapeText(pattern);
     }
 
-    @Override
-    public boolean convertPropertyToBoolean(String propertyKey) {
-        boolean booleanValue = false;
-        if (additionalProperties.containsKey(propertyKey)) {
-            booleanValue = Boolean.valueOf(additionalProperties.get(propertyKey).toString());
-        }
-
-        return booleanValue;
-    }
-
-    @Override
-    public void writePropertyBack(String propertyKey, boolean value) {
-        additionalProperties.put(propertyKey, value);
-    }
-
     /**
      * Output the Getter name for boolean property, e.g. isActive
      *
      * @param name the name of the property
      * @return getter name based on naming convention
      */
+    @Override
     public String toBooleanGetter(String name) {
         return booleanGetterPrefix + getterAndSetterCapitalize(name);
     }
 
-    @Override
-    public String sanitizeTag(String tag) {
-        tag = camelize(underscore(sanitizeName(tag)));
-
-        // tag starts with numbers
-        if (tag.matches("^\\d.*")) {
-            tag = "Class" + tag;
-        }
-        return tag;
-    }
-
```

Comment:
Would removing this cause issue with tag starting with numbers?

Reply:
because parent method (DefautCodeGen class) is actually doing exactly the same things so this overriding is useless

Reply:
Understood. If I remember correctly, the reason is that we don't want generators to use the default defined in the DefaultCodegen class as changing the default usually results in a lot of changes to at least a few generators. That's why we prefer to override the methods in the language generator class.

It's perfectly fine to remove it in this PR but later you may see us adding it back and slightly revising the logic to address certain edge cases.

Reply:
i understand the logic, thanks for the explanation

Reply:
anyway i will remove this change with your other review with another commit